docs/solution-guidance/Embedding-JavaScript-into-SharePoint.md

```diff
@@ -85,4 +85,4 @@ Files attached with JSLink are **not** re-executed on MDS navigation, use the As
 
 * [Wictor Wilén - The correct way to execute javascript functions in SharePoint MDS enabled sites](http://www.wictorwilen.se/the-correct-way-to-execute-javascript-functions-in-sharepoint-2013-mds-enabled-sites)
 * [Hugh Wood - SharePoint JavaScript context development - AsyncDeltaManager](https://rencore.com/blog/sharepoint-javascript-context-development-part-4-the-way-of-the-async-delta-manager/)
-* [Marc Anderson - Execute JavaScript after MDS (re)load](http://blog.symprogress.com/2013/09/sharepoint-2013-execute-javascript-function-after-mds-load/)
+* [Marc Anderson - Execute JavaScript after MDS (re)load](https://sympmarc.com/tag/javascript/)

```

Comment:
Remove this link... the target isn't correct for the description

Reply:
Changes applied, for some reasons is not showing in File changes UI.